src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java

```diff
@@ -26479,4 +26481,166 @@ private static final void registerNEIPlanetRecipes() {
+            Materials.Grade1PurifiedWater.getFluid(1000L),
+            Materials.Grade2PurifiedWater.getFluid(1000L),
+            Materials.Grade3PurifiedWater.getFluid(1000L),
+            Materials.Grade4PurifiedWater.getFluid(1000L),
+            Materials.Grade5PurifiedWater.getFluid(1000L),
+            Materials.Grade6PurifiedWater.getFluid(1000L),
+            Materials.Grade7PurifiedWater.getFluid(1000L),
+            Materials.Grade8PurifiedWater.getFluid(1000L)
+        };
+        for (int i = 0; i < purified_water.length; i++) {
+            GT_Values.RA.addCutterRecipe(
+                    ItemList.Circuit_Silicon_Ingot6.get(1L), // Optical Boule
+                    purified_water[i],
+                    ItemList.Circuit_Silicon_Wafer6.get((i + 1) * 2L),
+                    GT_Values.NI,
+                    wafer_duration_ticks *= 0.95,
+                    wafer_eu_per_tick);
+        }
+
+        Fluid oganesson = FluidRegistry.getFluid("oganesson") != null
+                ? FluidRegistry.getFluid("oganesson")
+                : FluidRegistry.getFluid("radon");
+
+        // Photonically Enhanced Wafer
+        GT_Values.RA.addLaserEngraverRecipe(
+                new ItemStack[] {
+                    ItemList.Circuit_Silicon_Wafer6.get(1L), // Photonically Prepared Wafer
+                    Materials.Glowstone.getNanite(1),
+                    GT_Utility.copyAmount(
+                            0L,
+                            GT_ModHandler.getModItem("supersolarpanel", "solarsplitter", 1L, 0)) // Solar Light Splitter
+                },
+                new FluidStack[] {Materials.Tin.getPlasma(1000L), new FluidStack(oganesson, 4000)},
+                new ItemStack[] {ItemList.Circuit_Silicon_Wafer7.get(1L)},
+                new FluidStack[] {Materials.Tin.getMolten(1000L)},
+                10 * 20,
+                7_864_320,
+                true);
+
+        Fluid solderUEV = FluidRegistry.getFluid("molten.mutatedlivingsolder") != null
+                ? FluidRegistry.getFluid("molten.mutatedlivingsolder")
+                : FluidRegistry.getFluid("molten.solderingalloy");
+
+        // Optically Perfected CPU
+        GT_Values.RA.addCircuitAssemblerRecipe(
+                new ItemStack[] {
+                    ItemList.Circuit_Chip_Optical.get(1L),
+                    ItemList.Optical_Cpu_Containment_Housing.get(1L),
+                    GT_OreDictUnificator.get(OrePrefixes.screw, Materials.Infinity, 16L)
+                },
+                new FluidStack(solderUEV, 288),
+                ItemList.Optically_Perfected_CPU.get(1L),
+                20 * 20,
+                1_966_080,
+                true);
+
+        // Optical CPU Containment Housing
+        GT_Values.RA.addFormingPressRecipe(
+                new ItemStack[] {
+                    ItemList.Circuit_Board_Optical.get(1L),
+                    GT_ModHandler.getModItem("miscutils", "itemFoilBotmium", 1L),
```

Comment:
can do Loader.isModLoader("miscutils") and then use ALLOYS (from GT++, it should be in as a compile only)
GT_OreDictUnificator(OrePrefixes.foil, ALLOYS.Botmium, 1)